configuration/pih/pih-config-haiti.json

```diff
@@ -70,6 +70,18 @@
           "mothersName": {
             "required": true
           }
+        },
+        "contactInfo": {
+            "phoneNumber": {
+              "regex": "(^\\d{8}$)|(^\\d{4}(?:\\)|[-|\\s])?\\s*?\\d{4}$)"
```

Comment:
I just combined the two regex from the zl.js.  I tested and it appears to work, but @louidorjp @rjacques89  would be good to review.